src/main/java/kitchenpos/domain/Order.java

```diff
@@ -1,63 +1,104 @@
 package kitchenpos.domain;
 
+import kitchenpos.domain.type.OrderStatus;
+import org.springframework.data.annotation.CreatedDate;
+
+import javax.persistence.*;
 import java.time.LocalDateTime;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
+
+import static kitchenpos.constants.ErrorCodeType.*;
 
+@Entity
+@Table(name = "orders")
 public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long orderTableId;
-    private String orderStatus;
-    private LocalDateTime orderedTime;
-    private List<OrderLineItem> orderLineItems;
 
-    public Order() {
+    @ManyToOne(fetch = FetchType.LAZY)
+    private OrderTable orderTable;
+
+    @Enumerated(EnumType.STRING)
+    private OrderStatus orderStatus;
+
+    @CreatedDate
+    private LocalDateTime orderedTime = LocalDateTime.now();
+
+    @Embedded
+    private OrderLineItems orderLineItems = new OrderLineItems();
+
+    protected Order() {
+    }
+
+    public Order(OrderTable orderTable, OrderStatus orderStatus) {
+        this.orderTable = orderTable;
+        this.orderStatus = orderStatus;
+    }
+
+    public Order(OrderTable orderTable, OrderStatus orderStatus, OrderLineItems orderLineItems) {
+        this.orderTable = orderTable;
+        this.orderStatus = orderStatus;
+        this.orderedTime = LocalDateTime.now();
+        this.orderLineItems = orderLineItems;
     }
 
-    public Order(Long id, Long orderTableId, String orderStatus, LocalDateTime orderedTime, List<OrderLineItem> orderLineItems) {
+    public Order(Long id, OrderTable orderTable, OrderStatus orderStatus, OrderLineItems orderLineItems) {
         this.id = id;
-        this.orderTableId = orderTableId;
+        this.orderTable = orderTable;
         this.orderStatus = orderStatus;
-        this.orderedTime = orderedTime;
+        this.orderedTime = LocalDateTime.now();;
         this.orderLineItems = orderLineItems;
     }
 
-    public Long getId() {
-        return id;
+    public void validCheckOrderStatusIsCookingAndMeal() {
+        if (Arrays.asList(OrderStatus.COOKING, OrderStatus.MEAL).contains(this.orderStatus)) {
+            throw new IllegalArgumentException(COOKING_MEAL_NOT_UNGROUP.getMessage());
+        }
```

Comment:
```suggestion
    public void validUngroupable() {
        if (Arrays.asList(OrderStatus.COOKING, OrderStatus.MEAL).contains(this.orderStatus)) {
            throw new IllegalArgumentException(COOKING_MEAL_NOT_UNGROUP.getMessage());
        }
```
이런식으로 목적에 맞는 메시지를 사용하는건 어때요? 🤔